### ISSUE TITLE
Put Download Link in the Readme

### ISSUE THREAD

**Issue by marmundo** [fibasile/MacReFlower#4]

**Comment by aonez** [comment]
Better to add the compilations in the [Releases](https://github.com/fibasile/MacReFlower/releases) section.

**Comment by lothar-cell** [comment]
I might be a tart, but couldn't find the download link, the zip hosted here contains bits and pieces, but not the actual app. Any chance for a link from someone?

**Comment by thetechnaddict** [comment]
I, being another tart, would really appreciate a compilation in the releases section. I am having to learn about this development environment through working through my mistakes in order to convert a pdf to an ebook... Or of course I could use the command line utility whist waiting for a release...

**Comment by alehandrof** [comment]
Yes, a downloadable file would preferable. For the moment, I just used the CLI utility with the settings from here:
https://github.com/fibasile/MacReFlower/blob/a21ac898b68ebab59b63f43c5c9f49c0e93513e0/MacReFlower/MRFAppDelegate.m#L211

**Comment by ghost** [comment]
As a workaround: It was actually not that hard to build the app on my mac (osx 10.11.6 el capitan with xcode 8.1). I just had to [download a zip](https://github.com/fibasile/MacReFlower/archive/master.zip) of the project, unzip it and load the whole folder into xcode. In order to build it, I had to disable code signing in the project settings (click  on the Project file in the root in the menu left, pick the settings for the target and go to to the build settings and under Signing and there under Code Signing Identity "Don't Code sign").

**Comment by ptbrowne** [comment]
Yes I used openmedi instructions and then used Product > Archive to have a file that I could put into "Applications". Thanks !
